src/main/java/com/factset/sdk/eventdriven/factsettrading/model/OrderSubscriptionRequest.java

```diff
@@ -3,21 +3,28 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.util.ArrayList;
-import java.util.List;
 
 @Data
 @NoArgsConstructor
 public class OrderSubscriptionRequest {
     Meta meta = new Meta("OrderSubscriptionRequest");
     SubscriptionData data = new SubscriptionData();
 
-    public OrderSubscriptionRequest(List<String> subscribeTo) {
-        this.data.subscribe.addAll(subscribeTo);
+    public OrderSubscriptionRequest(Subscribe subscribeTo) {
+        this.data.subscribe = subscribeTo;
     }
 
     @Data
     public static class SubscriptionData {
-        List<String> subscribe = new ArrayList<>();
+        Subscribe subscribe = new Subscribe();
+    }
+
+    @Data
+    public static class Subscribe{
+        boolean inboundOrders;
```

Comment:
I suggest adding a [`@Builder`](https://projectlombok.org/features/Builder) annotation on this as well.
Instead of having to do the good-old setter chain, a user can then use the fluent builder interface like in this example:

https://github.com/factset/enterprise-sdk-eventdriven-factsettrading-java/blob/33b07fc42038e34875dfd8b85cce40d5f30cd97f/README.md#L113-L117

Reply:
Used the @Builder annotation and updated the readme file